reference.go

```diff
@@ -24,6 +24,29 @@
 //	digest-hex                      := /[0-9a-fA-F]{32,}/ ; At least 128 bit digest value
 //
 //	identifier                      := /[a-f0-9]{64}/
+//
+// # Supported algorithms
+//
+// Implementations must register the algorithms they want to support for digests
+// by importing the appropriate implementation in a non-test file.
+//
+// For example, to register the sha256 algorithm implementation from Go's stdlib:
+//
+//	import (
+//		_ "crypto/sha256"
+//	)
+//
+// Even though digests may be assemblable as a string, always verify your input
+// with [digest.Parse] or use [digest.Digest.Validate] when accepting untrusted
```

Comment:
I believe referencing another package's godoc is done by using the package full name so in this case it would be something like `[github.com/opencontainers/go-digest.Parse]`?

At least that's what I understood from this [reference](https://tip.golang.org/doc/comment#doclinks)

Reply:
Whoops; forgot about this one; I just tried with pkgsite, and it looks like it works as expected; it recognises the short name, and picks it up from the import. When running locally, links go to;

- http://localhost:6060/github.com/opencontainers/go-digest#Parse
- http://localhost:6060/github.com/opencontainers/go-digest#Digest.Validate

So when hosted on `pkg.go.dev` would be https://pkg.go.dev/github.com/opencontainers/go-digest#Parse

<img width="896" alt="Screenshot 2024-06-06 at 00 48 03" src="https://github.com/distribution/reference/assets/1804568/5032b34f-7538-4fcb-99cb-6f066608413e">

Reply:
Yep, just tried it and it's working indeed. It appears I misunderstood the documentations.